src/test/java/core/service/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,56 @@
+package core.service.impl;
+
+import core.service.ReaderService;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String NON_EXISTING_FILENAME = "RAW_filename.csv";
+    private static final String EMPTY_FILENAME = "";
+    private static final String CORRECT_INPUT = "src/test/resources/testFile.csv";
+    private static final String EMPTY_FILE = "src/test/resources/testEmpty.csv";
+    private static final String WRONG_INPUT = "src/test/resources/test_file_wrong_data.csv";
+    private ReaderService fileReader = new FileReaderImpl();
```

Comment:
Implementation should be in BeforeClass method

Reply:
I think it's not necessary, but ok